Godeps/Godeps.json

```diff
@@ -2,371 +2,18 @@
 	"ImportPath": "github.com/coreos/rkt",
 	"GoVersion": "go1.5",
 	"Packages": [
-		"./...",
-		"github.com/appc/spec/actool",
-		"github.com/appc/cni/plugins/ipam/dhcp",
-		"github.com/appc/cni/plugins/ipam/host-local",
-		"github.com/appc/cni/plugins/main/bridge",
-		"github.com/appc/cni/plugins/main/ipvlan",
-		"github.com/appc/cni/plugins/main/macvlan",
-		"github.com/appc/cni/plugins/main/ptp",
-		"github.com/appc/cni/plugins/meta/flannel",
-		"github.com/appc/spec/ace"
```

Comment:
Sorry about that, definitely by mistake. Had to fight with godeps, and godeps won.